File: dart/8kyu/bool_to_word/bin/bool_to_word.dart
import "package:test/test.dart";

void main() {
  test('Tests', () {
    expect(bool_to_word(true), equals("Yes"));
    expect(bool_to_word(false), equals("No"));
  });
}

String bool_to_word(bool boolean) {
  return boolean ? 'Yes' : 'No';
}

File: typescript/8kyu/bool_to_word/bool_to_word.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.boolToWord = void 0;
const boolToWord = (bool) => {
    return bool ? "Yes" : "No";
};
exports.boolToWord = boolToWord;
const chai_1 = require("chai");
describe("boolToWord", function () {
    it("Sample tests", function () {
        chai_1.assert.equal((0, exports.boolToWord)(true), "Yes");
        chai_1.assert.equal((0, exports.boolToWord)(false), "No");
    });
});
